azext_iot/central/services/device.py

```diff
@@ -43,14 +47,27 @@ def get_device(
     url = "https://{}.{}/{}/{}".format(app_id, central_dns_suffix, BASE_PATH, device_id)
     headers = _utility.get_headers(token, cmd)
 
-    response = requests.get(url, headers=headers, verify=not should_disable_connection_verify())
+    # Construct parameters
+    query_parameters = {}
+    query_parameters["api-version"] = api_version
+
+    response = requests.get(url, headers=headers, params=query_parameters)
```

Comment:
Removal of `verify=not should_disable_connection_verify()` ?

Reply:
fixed